### DISCUSSION TITLE
Can't detect my http_user

### DISCUSSION THREAD

**Discussion by Ange7** [deployphp/deployer#2368] [Help needed] [answered]
Hello ! 

I have this error at the end of my deploy : 

```
In writable.php line 33:
                                              
  Can't detect http user name.                
  Please setup `http_user` config parameter
```

And when i run 
`ps axo comm,user | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | sort | awk '{print $2}' | uniq`
i have no result (empty line) 

my webserver (nginx) is not well configured ? 

Thank you

**Comment by antonmedv** [comment]
I don’t know anything about your nginx. Better check it out. You just need to set correct user of it.

**Comment by Ange7** [comment, marked as answer]
it's because of « hidepid » service conf on my system, maybe it can be help someone else.
